filebeat/beater/filebeat.go

```diff
@@ -48,10 +51,10 @@ func (fb *Filebeat) Run(b *beat.Beat) error {
 	var wgEvents *sync.WaitGroup // count active events for waiting on shutdown
 	var finishedLogger publisher.SuccessLogger
 
-	if fb.config.ShutdownTimeout > 0 {
-		wgEvents = &sync.WaitGroup{}
-		finishedLogger = newFinishedLogger(wgEvents)
-	}
+	//if fb.config.ShutdownTimeout > 0 || *once {
```

Comment:
@urso do you see an issue with this? As long as we don't use wgEvents nothing will happen.